docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -141,12 +141,12 @@ Then, load your DataContext into memory using the `get_context()` method.
 
 Next, load the following Datasource configuration that will connect to data in GCS,
 
-```python file=../../tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs_yaml_configs.py#L201-L219
+```python file=../../tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_gcs_yaml_configs.py#L206-L224
 ```
```

Comment:
My hero 🙇🏽 appreciate you doing this monotonous but important work to align the docs under test.